packages/mui-material-nextjs/package.json

```
@@ -0,0 +1,85 @@
+{
+  "name": "@mui/material-nextjs",
+  "version": "5.14.18",
+  "private": false,
+  "author": "MUI Team",
+  "description": "A tiny wrapper for Material UI integration with Next.js.",
```

```suggestion
  "description": "Collection of utilities for integration between Material UI and Next.js.",
```

It's not really a wrapper around anything.